entrypoint/versionbump/componentsbump_test.go

```diff
@@ -14,20 +14,16 @@ import (
 )
 
 const (
-	flannelRke2       = "kubectl get node -o yaml : | grep 'hardened-flannel' -A1, "
-	flannelK3s        = "/var/lib/rancher/k3s/data/current/bin/flannel, "
-	calico            = "kubectl get node -o yaml : | grep 'hardened-calico' -A1, "
-	ingressController = "kubectl get node -o yaml : | grep 'nginx-ingress-controller' -A1, "
-	corednsRke2       = "kubectl get node -o yaml : | grep 'hardened-coredns' -A1, "
-	coreDnsk3s        = "kubectl get node -o yaml : | grep 'mirrored-coredns' -A1, "
-	metricsServer     = "kubectl get node -o yaml : | grep 'metrics-server' -A1, "
-	etcdRke2          = "kubectl get node -o yaml : | grep 'hardened-etcd' -A1, "
-	containerd        = "kubectl get node -o yaml : | grep  containerd, "
+	kgn               = "kubectl get node -o yaml"
+	canalFlannel      = kgn + " : | grep 'hardened-flannel' -A1, "
```

Comment:
We can but not sure if we are using them anywhere else... also this allows scoping these vars within the test

Reply:
we can use the `kgn`  like this in other tests like cilium:
`						Cmd:                  kgn + " : | grep mirrored-cilium  -A1,kubectl get node -o yaml : | grep hardened-cni-plugins -A1",`

so if you want to use like this we dont need even to change this anywhere else, because it still visible for all other tests on this pkg

Reply:
but tbh not seeing any huge gain on that , since those commands are not stuff we are going to change regularly and so on

Reply:
I don't have a _super_ strong opinion here, but I do kind of like the explicit `kubectl get node` since it's easy for someone who is less familiar with the commands we use and the processes to understand what is being run exactly if they look at the code.